README.md

```diff
@@ -150,6 +150,15 @@ The fix is to update /etc/sysconfig/docker to ensure that minikube's environment
 
 Remember to turn off the imagePullPolicy:Always, as otherwise kubernetes won't use images you built locally.
 
+#### Enabling Docker Insecure Registry
+
+Minikube allows users to configure the docker engine's `--insecure-registry` flag. You can use the `--insecure-registry` flag on the
+`minikube start` command to enable insecure communication between the docker engine and registries listening to reqeusts from the CIDR range.
+
+One nifty hack is to allow the kubelet running in minikube to talk to registries deployed inside a pod in the cluster without backing them
+with TLS certificates. Because the service cluster IP range is known to be available at 10.0.0.0/24, users can pull images from registries

```

Comment:
Good point. I was referring to the CIDR range one would specify to `--insecure-registry`. I'll clarify that and make it more clear :)